roles/openshift_health_checker/openshift_checks/ovs_version.py

```diff
@@ -16,8 +16,8 @@ class OvsVersion(NotContainerizedMixin, OpenShiftCheck):
     tags = ["health"]
 
     openshift_to_ovs_version = {
-        "3.7": ["2.6", "2.7", "2.8"],
-        "3.6": ["2.6", "2.7", "2.8"],
+        "3.7": ["2.6", "2.7", "2.8", "2.9"],
+        "3.6": ["2.6", "2.7", "2.8", "2.9"],
```

Comment:
Seems a little weird to have a mapping for "3.7" in the release-3.6 branch, but it shouldn't hurt anything.

Reply:
That's what I get for cherry-picking